go/pkg/zen/README.md

```
@@ -90,8 +91,9 @@ func main() {
 	createUserRoute := zen.NewRoute("POST", "/users", func(ctx context.Context, s *zen.Session) error {
 		// Parse request body
 		var req CreateUserRequest
-		if err := s.BindBody(&req); err != nil {
-			return err // This will be handled by error middleware
+		req, err := zen.BindBody[CreateUserRequest](s)
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)